### ISSUE TITLE
onMouseMove stops propagation of touch events

### ISSUE THREAD

**Issue by hirasso** [paperjs/paper.js#585]
Hi there, love your work!! I just came over a problem: If I use the onMouseMove function, all the touch-events will be consumed as well, which prevents native functionality like e.g. scrolling. I tried to only use the onMouseMove event for desktops like this:
```javascript
$.support.touch = 'ontouchstart' in window;
console.log( $.support.touch ); // logs 'true' on my ipad, tested
if( !$.support.touch ) {
    // this should only run if on a non-touch device, but it keeps running everywhere
    function onMouseMove(event) {
        // code
    }
}
```
but even this didn't prevent the mouse events from being consumed on my touch device. The only fix I found for this is to include two duplicate scripts for desktop and touch devices respectively, with the mouse functionality being disabled in the mobile version of the script. Do you have any better suggestions?

**Comment by lqez** [comment]
I'm also in trouble with this issue. :$

**Comment by lehni** [comment]
#686 appears to be related, and there is a proposed fix.

**Comment by lehni** [comment]
I believe I have fixed #686 now. @hirasso could you please check if this also finally solves your issue here?

**Comment by lehni** [comment]
From all my tests, this is fixed now, see #686 for details. Feel free to reopen if it's still a problem.
